### PR TITLE
drop RedHat and CentOS 7 & 8 support

### PR DESCRIPTION
**Pull request by TheMeier**
https://voxpupuli.org/blog/2024/05/08/CentOS-7-CentOS-Stream-8-deprecation/

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -32,16 +32,13 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "7",
         "8",
```
Comment (bastelfreak) [inline metadata.json:35]: IMO RedHat 8 should be dropped as well

Comment (TheMeier) [reply metadata.json:35]: ok... also Alma and Rocky v8 then?

File: metadata.json [hunk 2]
```diff
@@ -32,16 +32,12 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "7",
-        "8",
```
Comment (bastelfreak) [inline metadata.json:36]: sorry for the noise. after some back and forth we will keep redhat 8 and only drop redhat 7. can you please add 8 back?

Comment (TheMeier) [reply metadata.json:36]: done

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**